ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -355,7 +355,7 @@ describe('ExpandedApp', () => {
       appType: AppType.RSTUDIO,
       status: AppStatus.RUNNING,
       proxyUrls: {
-        'rstudio-service': proxyUrl,
```

Comment:
would be nice to have a constant for this

Reply:
I'm down for that..do u have any suggestion as to where I should put it? any existing examples?

Reply:
there's a `constants` file but I'm not sure if that's appropriate... maybe someplace like an analysis-utils file (or analysis/utils?)